backend/addcorpus/json_corpora/import_json.py

```diff
@@ -0,0 +1,282 @@
+    default_sort = get_path(data, 'options', 'default_sort') or {}
+    language_field = get_path(data, 'options', 'language_field') or ''
+    document_context = get_path(data, 'options', 'document_context') or {}
+    source_data = get_path(data, 'source_data')
+    return CorpusConfiguration(
+        title=title,
+        description=description,
+        category=category,
+        es_index=es_index,
+        languages=languages,
+        min_date=min_date,
+        max_date=max_date,
+        default_sort=default_sort,
+        language_field=language_field,
+        document_context=document_context,
+        source_data=source_data
+    )
+
+
+def _parse_date(date: str):
+    return datetime.strptime(date, '%Y-%m-%d').date()
+
+
+def _import_fields(data: Dict, configuration: CorpusConfiguration) -> None:
+    fields_data = get_path(data, 'fields')
+
+    for field_data in fields_data:
+        field = _parse_field(field_data)
+        field.corpus_configuration = configuration
+        field.full_clean()
+        field.save()
+
+    _include_ngram_visualisation(configuration.fields.all())
+
+
+def _parse_field(field_data: Dict) -> Field:
+    name = get_path(field_data, 'name')
+    display_name = get_path(field_data, 'display_name')
+    description = get_path(field_data, 'description')
+    results_overview = get_path(field_data, 'options', 'preview')
+    hidden = get_path(field_data, 'options', 'hidden')
+    extract = get_path(field_data, 'extract')
+
+    field = Field(
+        name=name,
+        display_name=display_name,
+        description=description,
+        results_overview=results_overview,
+        hidden=hidden,
+        csv_core=results_overview,
+        extract_options=extract
+    )
+
+    field_type = get_path(field_data, 'type')
+    parsers = {
+        'text_content': _parse_text_content_field,
+        'text_metadata': _parse_text_metadata_field,
+        'url': _parse_url_field,
+        'integer': _parse_numeric_field,
+        'float': _parse_numeric_field,
+        'date': _parse_date_field,
+        'boolean': _parse_boolean_field,
+        'geo_json': _parse_geo_field,
+    }
+    field = parsers[field_type](field, field_data)
+
+    return field
+
+
+def _parse_text_content_field(field: Field, field_data: Dict) -> Field:
+    language = _parse_language(field_data)
+    has_single_language = language and language != 'dynamic'
+
+    field.es_mapping = es_mappings.main_content_mapping(
+        token_counts=True,
+        stopword_analysis=has_single_language,
+        stemming_analysis=has_single_language,
+        language=language if has_single_language else None,
+    )
+    field.language = language
+    field.display_type = 'text_content'
+    field.search_filter = {}
+    field.searchable = True
+    field.search_field_core = True
+
+    visualize = get_path(field_data, 'options', 'visualize')
+    if visualize:
+        field.visualizations = [
+            VisualizationType.WORDCLOUD.value
```

Comment:
Why is this set to WORDCLOUD now?

Reply:
Yes, it's a boolean: https://github.com/UUDigitalHumanitieslab/I-analyzer/blob/1dfa42b8aafab54108fa8e03bb43a7ace6527be7/backend/addcorpus/schemas/corpus.schema.json#L140-L143